src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/ReadThreadingAssemblerArgumentCollection.java

```diff
@@ -23,7 +24,11 @@
      */
     @Advanced
     @Argument(fullName="kmerSize", shortName="kmerSize", doc="Kmer size to use in the read threading assembler", optional = true)
-    public List<Integer> kmerSizes = Arrays.asList(10, 25);
+    public List<Integer> kmerSizes = new ArrayList<>();
```

Comment:
This is more annoying that it should be.  I thought that registering an 'AllelesAsListSerializer' would let 'Kryo' work with these.  However, spark has a number of hardcoded serializers that override the user's choices.  These include a buggy version of 'AllelesAsListSerializer' that is incapable of serializing primitive arrays.  (See [KryoSerializer line 136](https://github.com/apache/spark/blob/master/core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala#L136))  I filed a bug report with Chill (https://github.com/twitter/chill/issues/255), but I think it will be a long time before this is addressed and available  (since it would mean reving the chill version in spark...)